### ISSUE TITLE
[BUG] Using better_player: 0.0.76 under Flutter 2.2.3 compilation failed

### ISSUE THREAD

**Issue by dev-zhang** [jhomlala/betterplayer#731]
It looks like better_player: 0.0.76 depends on Flutter 2.5, but I don't see this version constraint in the plugin.
You'd better update the version constraint in the plugin for Flutter and Dart, thanks.

**Comment by jhomlala** [comment]
@dev-zhang Yes you're right, I will add constraint in next version.

**Comment by jamieownsu** [comment]
I'm getting this error when compiling for ios in 0.0.76 / Flutter 2.5.1 / xcode 13:
```
    /Users/jamieotoole/.pub-cache/hosted/pub.dartlang.org/better_player-0.0.76/ios/Classes/BetterPlayer.m:6:9: fatal error: 'better_player/better_player-Swift.h' file not found
    #import <better_player/better_player-Swift.h>
```

**Comment by jhomlala** [comment]
@jamieownsu This is not connected with the issue author topic.

**Comment by jhomlala** [comment]
Fixed in 0.0.77.
